docs/using/plugins.md

```diff
@@ -1,65 +1,88 @@
 # Creating custom interactions via plugins
 
-Custom interactions (i.e. non-12-6 Lennard-Jones or non-harmonic bond or angle terms) can be introduced into the OpenFF stack via plugin interfaces. For each type of physical interaction in the model, plugins subclassed from `ParameterHandler` and `SMIRNOFFCollection` must be defined and exposed in the entry points `openff.toolkit.plugins.handlers` and `openff.interchange.plugins.collections`respectively.
+Custom interactions (i.e., interactions other than 12-6 Lennard-Jones, harmonic bonds, or harmonic angles) can be introduced into the OpenFF stack via plugin interfaces. To create a plugin, create a subclass of both [`ParameterHandler`] and [`SMIRNOFFCollection`] for each new type of physical interaction in the model, then expose them to Interchange via the `openff.toolkit.plugins.handlers` and `openff.interchange.plugins.collections` [SetupTools entry points], respectively.
 
-Currently, these custom interactions can only be exported to OpenMM. There are two routes for this export: one using existing Interchange machinery and another that allows for completely custom behavior.
+Parameter handlers are responsible for the system- and engine-independent aspects of parametrization; in other words, they define the force field itself. By contrast, a SMIRNOFF collection handles the parametrization of a particular system and the preparation of that system for simulation with an external MD engine. The base classes for parameter handlers are defined in the OpenFF Toolkit, while collections are defined in Interchange.
```

Comment:
Since the first paragraph introduces something new and this is a helpful back-step to clarify the existing infrastructure, it'd be nice to communicate "the OpenFF stack ships with everything SMIRNOFF already, you're only responsible for writing the custom stuff you want to tack on." The last sentence has some implication of the first clause but I think I don't think that's the main message. I couldn't find a way to add this in without being repetitive